### ISSUE TITLE
Move IMPORTANT NOTE ABOUT EMAIL VALIDATION where it belongs!

### ISSUE THREAD

**Issue by blaasvaer** [jquery-validation/jquery-validation#2162]
Please, put the `IMPORTANT NOTE ABOUT EMAIL VALIDATION` from the README in a place that make people actually see it. There seems to be SOMETHING wrong since a NUMBER of people are missing this very important note.

**Comment by blaasvaer** [comment]
When I as a developer land on this page by a simple search: [https://jqueryvalidation.org/email-method/](https://jqueryvalidation.org/email-method/), I see this line:

> In case you need a custom email pattern use the $.validator.methods property.

Now, as a developer I assume that a 'normal' valid e-mail (don't go yadda yadda about @localhost being a valid e-mail – we're talking real world implementations here) consists of:

```
some_character, an at-sign, some more character, a dot and then at least 2 characters more.
```

So, when reading `In case you need a custom email …`, I think `hell no, I just need at normal e-mail validation, not a custom one`. And there you have it; real world developers facing real world documentation ... 'causing this thread.

**Comment by Arkni** [comment]
Hi @blaasvaer,

Do you think it would be a good idea to add the whole warning to the documentation of `email` method?

**Comment by Arkni** [comment]
Your assumption is correct. I will update the docs as soon as I have some time.

**Comment by Arkni** [comment]
I'll take care of this soon

**Comment by blaasvaer** [comment]
Nice

**Comment by Arkni** [comment]
This should be fixed by jquery-validation/validation-content#47
